BridgeSDK/BridgeAPI/SBBBridgeInfoProtocol.h

```diff
@@ -66,9 +66,25 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly) SBBEnvironment environment;
 
 /**
- App group identifier used for the suite name of NSUserDefaults, and for the name of the shared
- container, which is used both to configure the background session and as the place to store
- temporary copies of files being uploaded to Bridge (if provided).
+ Tells the Bridge libraries to use the standard user defaults suite.
+ 
+ @note This flag is intended only for backward compatibility when upgrading apps built with older versions of Bridge libraries that used the standard user defaults suite. It will be ignored in any case if either userDefaultsSuite or appGroupIdentifier are set.
+ */
+@property (nonatomic, readonly) BOOL useStandardUserDefaults;
```

Comment:
yes, if we want them to keep what they had stored in NSUserDefaults. I just think that by default, going forward, we ought not to be using standard defaults, so we can manage ours more easily without interfering with settings we don't know or care about that the app developer uses.